app/views/markdown_pages/discord.html.erb

```diff
@@ -16,7 +16,15 @@
 <center><%= image_tag 'https://discordapp.com/api/guilds/453373412707008522/widget.png' %></center>
 <br>
 
+<% if @page.present? %>
   <%= render 'markdown_pages/markdown_page', page: @page, page_title: @page_title %>
+<% else %>
+  <p>There is no content to show currently</p>
```

Comment:
@amanpatel123 We don't need this for this view. 
<img width="1095" alt="Screen Shot 2020-05-16 at 11 12 48 AM" src="https://user-images.githubusercontent.com/46115782/82123360-ff113e80-9766-11ea-8950-88906ddc6ed9.png">

Reply:
okay! will remove it